packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -167,6 +175,76 @@ class MemoryBodyState extends State<MemoryBody>
       });
     });
 
+    addAutoDisposeListener(eventChartController.tapNotifier, () {
+      if (!eventChartController.tapNotifier.value.isEmpty &&
+          hoverOverlayEntry != null) {
+        hideHover();
+      }
+      if (eventChartController.tapNotifier.value.tapDownDetails != null) {
+        final tapData = eventChartController.tapNotifier.value;
+        final index = tapData.index;
+        final timestamp = tapData.timestamp;
+
+        final copied = TapNotifier.copy(eventChartController.tapNotifier.value);
+        vmChartController.setTapNotifier(copied);
+        androidChartController.setTapNotifier(copied);
+
+        final allValues = ChartsValues(this, index, timestamp);
+        if (isDebugging) {
+          print('Event Chart TapNotifier '
+              '${JsonUtils.prettyPrint(allValues.toJson())}');
+        }
+        showHover(context, allValues, tapData.tapDownDetails.globalPosition);
+      }
+    });
+
+    addAutoDisposeListener(vmChartController.tapNotifier, () {
+      if (!vmChartController.tapNotifier.value.isEmpty &&
+          hoverOverlayEntry != null) {
+        hideHover();
+      }
+      if (vmChartController.tapNotifier.value.tapDownDetails != null) {
+        final tapData = vmChartController.tapNotifier.value;
+        final index = tapData.index;
+        final timestamp = tapData.timestamp;
+
+        final copied = TapNotifier.copy(vmChartController.tapNotifier.value);
+        eventChartController.setTapNotifier(copied);
+        androidChartController.setTapNotifier(copied);
+
+        final allValues = ChartsValues(this, index, timestamp);
+        if (isDebugging) {
```

Comment:
why not use debugLogger instead of isDebugging and print?

Reply:
Wanted an isDebugging (that only shows up if turned on).  I've added a field just for that with a test that ensures isDebugging is never checked in as true.